modules/tinymce/changelog.txt

```diff
@@ -22,12 +22,12 @@ Version 5.3.0 (TBD)
     Fixed supplementary special characters being truncated when inserted into the editor. Patch contributed by mlitwin. #TINY-4791
     Fixed toolbar buttons not set to disabled when the editor is in readonly mode #TINY-4592
     Fixed the editor selection incorrectly changing when removing caret format containers #TINY-3438
-    Fixed bug where title, width, height would be set to empty string values when updating an image and removing those using the image dialog #TINY-4786
+    Fixed bug where title, width, or height would be set to empty string values when updating an image and removing those attributes using the image dialog #TINY-4786
     Fixed `ObjectResized` event firing when an object wasn't resized #TINY-4161
     Fixed `ObjectResized` and `ObjectResizeStart` events incorrectly fired when adding or removing table rows and columns #TINY-4829
     Fixed the placeholder not hiding when pasting content into the editor #TINY-4828
     Fixed an issue where the editor would fail to load if local storage was disabled #TINY-5935
-    Fixed an issue where an uploaded image would reuse a cached image with a different mime type #TINY-5988
+    Fixed an issue where an uploaded image would reuse a cached image with a the same filename but different mime type #TINY-5988
```

Comment:
This changed the meaning and is incorrect now.

Reply:
reverted

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5671/commits/3afd8db191c5d5fa0ad10ebcd1665364c10dcd66